Source/PLCrashReportThreadInfo.m

```diff
@@ -26,7 +26,10 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
+#ifndef PLCRASH_REPORT_THREAD_INFO
+#define PLCRASH_REPORT_THREAD_INFO
 #import "PLCrashReportThreadInfo.h"
+#endif
```

Comment:
Wrong guard usage. It should be only inside headers. Not in placed where you import it